### ISSUE TITLE
Support the cdk version 0.34.0

### ISSUE THREAD

**Issue by cohalz** [aereal/cdk-mackerel-container-agent#59]
The released npm package only supports up to 0.31.0.

```
npm WARN @aereal/cdk-mackerel-container-agent@0.4.0 requires a peer of @aws-cdk/aws-ecs@^0.27.0 || ^0.28.0 || ^0.29.0 || ^0.30.0 || ^0.31.0 but none is installed. You must install peer dependencies yourself. 
npm WARN @aereal/cdk-mackerel-container-agent@0.4.0 requires a peer of @aws-cdk/cdk@^0.27.0 || ^0.28.0 || ^0.29.0 || ^0.30.0 || ^0.31.0 but none is installed. You must install peer dependencies yourself. 
```

**Comment by cohalz** [comment]
@aereal  What about this?

**Comment by aereal** [comment]
shipped as 0.5.0 https://github.com/aereal/cdk-mackerel-container-agent/commit/c4246fe2b0c6652efc69e91cf0f13057c57fa6de
